sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/TestMethodShouldContainAssertion.cs

```diff
@@ -36,6 +36,8 @@ public sealed class TestMethodShouldContainAssertion : SonarDiagnosticAnalyzer
     {
         internal const string DiagnosticId = "S2699";
```

Comment:
optional (clean as you code): 
- DiagnosticId can be private
- IsTestIgnored can be private